CustomLookup/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
There's already a gitignore at the root of the repo, you can remove this one.

Reply:
Removed